crtauth/ssh.py

```diff
@@ -52,7 +56,7 @@ def base64url_decode(s):
     except:
         _, e, tb = sys.exc_info()
         raise exceptions.InvalidInputException(
```

Comment:
I remember there being a good reason for passing on the traceback as a parameter to raise. Can you grok why? I'm guessing there is a reason we're not just simply adding `as e` to `except` but instead messing around with `sys.exc_info()`